### ISSUE TITLE
Auto generated defined_tags result into recreation of resource

### ISSUE THREAD

**Issue by neermitt** [oracle-terraform-modules/terraform-oci-oke#305]
module: policies

```hcl
resource "oci_identity_policy" "operator_instance_principal_dynamic_group" { 
 provider       = oci.home 
 compartment_id = var.tenancy_id
 description    = "policy to allow operator host to manage dynamic group"
 name           = var.label_prefix == "none" ? "operator-instance-principal-dynamic-group-${substr(uuid(), 0, 8)}" : "${var.label_prefix}-operator-instance-principal-dynamic-group-${substr(uuid(), 0, 8)}"
 statements     = ["Allow dynamic-group ${var.dynamic_group} to use dynamic-groups in tenancy"]
 count          = (var.oke_kms.use_encryption == true &amp;&amp; var.operator.bastion_enabled == true &amp;&amp; var.operator.operator_instance_principal == true) ? 1 : 0}
```

as `name` in the resource is using a `uuid` function, results into new uuid every execution and shows a diff  in plan even when nothing has changed in configuration.

**Comment by hyder** [comment]
Hi,

Sorry for the delay. We'll log a bug in the operator module.

**Comment by hyder** [comment]
Closed in #332
